recipes/so3/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

```suggestion
conan_basic_setup()
```

TARGETS generates CONAN_PKG:: namespace, but I doubt you are using it directly in your code.